### ISSUE TITLE
after upgrading to Theme Beta = Error - Perl execution failed

### ISSUE THREAD

**Issue by PauloBoaventura** [authentic-theme/authentic-theme#1503]
after upgrading to Beta, I got the following error:

Error - Perl execution failed
Undefined subroutine & main :: get_miniserv_config called at /usr/libexec/webmin/authentic-theme/session_login.cgi line 15.

I can't get into the panel, any solution?

**Comment by PauloBoaventura** [comment]
@iliarostovtsev How could I go back to the previous version?

**Comment by iliarostovtsev** [comment]
Hi,

You would want to use `update-theme.sh -h` script from theme's directory or reinstall Webmin package or download the latest version of the theme from releases page on GitHub and manually unpack it into `/usr/{libexec|share}/webmin` directory or apply [this](https://github.com/webmin/webmin/commit/35b4e4716f41d9e93c5548dbe30d96c0b8724cb9) patch for Webmin and restart Webmin as follows:

```
/etc/webmin/restart
```

**Comment by PauloBoaventura** [comment]
Hello, thanks for responding promptly ...
I restarted Webmin and saw several problems on google. To which you refer ... But I did not succeed.
I restarted the server and continued with the problem.

I didn't particularly know how to update via GitHub.

So my solution was to delete the directory:

rm - rf / usr / libexec / webmin / authentic-theme / *

and copy from another server with the same operating system ...

rsync -auv root@138.129.000.001: / usr / libexec / webmin / authentic-theme / * / usr / libexec / webmin / authentic-theme /
